docs/concepts/point-in-time-join.md

```diff
@@ -6,27 +6,29 @@ parent: Feathr Concepts
+Models should not use the future to predict the present. If done so, this is called feature leakage. For example, consider an observation data with a timestamp of 7 days ago. In the eyes of this observation data, 7 days ago is its present. Now imagine that time-series feature data is ingested into your data warehouse and it contains recent purchase data from 2 days ago. In the eyes of the observation data, all the newly ingested data during the last week are future features. If you blindly use the latest feature data in your model, you are leaking future data into it. This mistake happens more often than people think.
 
-The model will perform better during training(usually), but it will not perform as good during online serving. Why? There is no future data on online serving anymore.
+The model will (usually) perform better during training, but it will not perform as good during online serving. Why? There is no future data during online serving.
 
 ## Point-in-time Correctness
 
-Point-in-time correctness ensures that no future data is used for training.
+Point-in-time correctness ensures that no future data is used for training. It can be achieved via two approaches:
+- If your observation data has a global timestamp for all observation events, then you can simply time-travel your feature dataset back to that timestamp. 
+- If your observation data has different timestamps for each observation event, then you need to use point-in-time join for each event. 
 
-Point-in-time correctness can be achieved via two approaches. If your observation data has a global timestamp for all observation events, then you can simply time-travel your feature dataset back to that timestamp. If your observation data has different timestamps for each observation events, then you need to point-in-time join for each events. The first approach is easier to implement but have more restrictions(global timestamp). The second approach provides better flexibility and no feature data is wasted. Feathr uses the second approach and can scale to large datasets.
+The first approach is easier to implement but have more restrictions (global timestamp). The second approach provides better flexibility and no feature data is wasted. Feathr uses the second approach and can scale to large datasets.
 
 ## Point-in-time Feature Lookup in Feathr
 
 To illustrate the power of point-in-time feature lookup in Feathr, consider the following (dummy) scenario, where:
 
 - the label (observation) data spanned across multiple days
```

Comment:
@JustintyC , thank you a lot for providing the grammar enhancement. It could be better if you also do us a favor to unify the Capitalize words of each bullet point, e.g., `the` -> `The`.